starter-code/server.js

```diff
@@ -35,14 +36,16 @@ app.use(express.static('./public'));
 app.get('/new', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
   // Put your response here...
+  // 5, This is interacting with the Artcle.fetchAll function and is being called //
```

Comment:
Correct on 5! It's actually not interacting with any methods...and is the READ part of CRUD